finality-aleph/src/aggregation/mod.rs

```diff
@@ -0,0 +1,12 @@
+use crate::crypto::Signature;
+use aleph_bft::{rmc::Message, SignatureSet};
+use sp_runtime::traits::Block;
+
+mod aggregator;
+mod multicast;
+
+pub use aggregator::BlockSignatureAggregator;
+pub use multicast::{Hash, Multicast, Multisigned, SignableHash};
```

Comment:
Are those (under `multicast`) used outside of this module? If not they shouldn't be `pub` here.